week-4/A-objects-intro/exercise-part-1.js

```diff
@@ -6,4 +6,36 @@ can describe with a JavaScript object
 Assign each of them to a separate variable
 
 */
+var car = {
+  brand: "BMW",
+  color: "black",
+  Manual: false,
+  doors: 2,
+  year: 2017
+};
+var flat = {
+  floor: "1st floor",
+  rooms: 4,
+  gourden: false
+};
 
+var mobile = {
+  brand: "Iphone",
+  model: "6",
+  memore: 64,
+  touchScreen: true
+};
+
+var sport = {
+  sportType: "foot ball",
+  teamMembers: 11,
+  iternationalGeam: true
+};
+

```

Comment:
Good. More care is needed with spelling: `garden`, `memory`, `internationalTeam`...